platform/solr/solr-dependencies/pom.xml

```diff
@@ -21,199 +21,627 @@
     </parent>
     <artifactId>solr-dependencies</artifactId>
     <name>DDF :: Platform :: Solr :: Dependencies</name>
-    <description>Shaded jar of Solr core dependencies.</description>
-    <packaging>jar</packaging>
-    <properties>
-        <jts.version>1.15.0</jts.version>
-        <dropwizard.version>3.2.6</dropwizard.version>
-        <s2.version>1.0.0</s2.version>
-    </properties>
+    <description>Contains solr dependencies. Exports solr related packages required by DDF.</description>
+    <packaging>bundle</packaging>
     <dependencies>
         <dependency>
             <groupId>org.apache.solr</groupId>
             <artifactId>solr-core</artifactId>
-            <exclusions>
```

Comment:
❓ Does Intellij or the Maven dependency tree plugin show any conflicts for this module?  Version conflicts caused unit tests to fail to run in Intellij before.

http://jason4zhu.blogspot.com/2017/08/resolving-maven-dependency-conflict.html
https://maven.apache.org/plugins/maven-dependency-plugin/examples/resolving-conflicts-using-the-dependency-tree.html

Reply:
Yeah, there are a handful of conflicts, around 10. I've ran quite a few unit tests from intellij ranging from platform to the catalog and didn't experience any problems though. Were there any in specific that were causing problems?